sbt-cross/src/main/scala/sbtcross/CrossPlugin.scala

```diff
@@ -0,0 +1,24 @@
+package sbtcross
+
+import sbt._
+import StringUtilities.nonEmpty
+
+import scala.language.implicitConversions
+
+object CrossPlugin extends AutoPlugin {
+  override def requires: Plugins = plugins.JvmPlugin
+
```

Comment:
That tests has Native projects. I'm talking about a build that would only have *JVM* projects (or JVM projects and Native/JS projects, but no crossProject). Even in a JVM project when there is no crossProject in the build, `%%%` should work.